Classes/Service/LazyLoading/ProductsQueryDispatcher.php

```diff
@@ -152,7 +151,7 @@ protected function queryBuilderToSql(QueryBuilder $queryBuilder): string
         foreach ($queryBuilder->getParameters() as $key => $value) {
             // prefix array keys with ':'
             //all non numeric values have to be quoted
-            $queryParameters[':' . $key] = is_numeric($value) ? $value : "'$value'";
+            $queryParameters[':' . $key] = is_numeric($value) ? $value : "'${value}'";
```

Comment:
Looking at this part: `"'${value}'"`

I don't like variables in strings, no matter how they're put there.

Also it's being quoted for DB insertion, right? So it should be quoted using the query builder's escape function.

Reply:
They are technically quoted before, looks like the query is just "extracted" as a "subQuery".
Think we need to take a look at this and change it when we look at task #212 anyway.
Just fixed the curly syntax now.

Reply:
OK